include/constants/battle.h

```diff
@@ -163,7 +176,9 @@ enum Terrain {
 #define STATUS2_MEAN_LOOK                   (1 << 26)
```

Comment:
I'm going to be irritating and ask if there's another name that can be used other than STATUS2

Reply:
volatile is for burn etc, not sure what the other status effect terminology is unless you want STATUS_NON_VOLITILE_MEAN_LOOK everywhere which is arguably more annoying

Reply:
ehhh note to self: come back to this later